sdcm/nemesis.py

```diff
@@ -2947,12 +2948,17 @@ def disrupt_remove_node_then_add_node(self):  # pylint: disable=too-many-branche
             # terminate node
             self._terminate_cluster_node(node_to_remove)
 
+        @retrying(n=3, sleep_time=5, message="Removing node from cluster...")
```

Comment:
maybe 15 seconds won't be enough in tests where the whole cluster + network will be very busy?

Reply:
also, shouldn't you here be catching the specific exception you are raising in case of failure below?

Reply:
The run_nodetool don't raise specific exception, So i left generic exception also